src/components/SourcesSimpleView.js

```diff
@@ -87,11 +83,13 @@ class SourcesSimpleView extends React.Component {
         ]
     );
 
-    sourceIsOpenShift = source => this.sourceTypeMap.get(source.source_type_id) === 'openshift';
+    sourceIsOpenShift = source => this.props.sourceTypes.find((type) => type.id === source.source_type_id).name === 'openshift';
     formatURL = source => source.endpoints && source.endpoints[0] && endpointToUrl(source.endpoints[0]);
 
-    applicationFormatter = apps => apps.map(a => this.appTypeMap.get(a.application_type_id)).join(', ');
-    sourceTypeFormatter = sourceType => (this.sourceTypeMap.get(sourceType) || sourceType || '');
+    applicationFormatter = apps =>
+        (apps.map(a => this.props.appTypes.find((type) => type.id === a.application_type_id).display_name).join(',\n'));
```

Comment:
`this.props.appTypes.find((type) => type.id === a.application_type_id).display_name` will crash if nothing is found.